changes/891.feature.md

```diff
@@ -0,0 +1,3 @@
+To make it possible to use events with decorators, where get_guild is required
```

Comment:
could we just reword this to something like "Added a get_guild (and possibly fetch_guild) shortcut method(s) to the voice events"